src/events/sdk-key-decoder.ts

```diff
@@ -1,27 +1,70 @@
 import { Base64 } from 'js-base64';
 
-const PATH = 'v0/i';
+const EVENT_INGESTION_HOSTNAME_KEY = 'eh';
+const CONFIGURATION_HOSTNAME_KEY = 'ch';
```

Comment:
@typotter Since I got here first 😉 , I selected `ch` (for `configuration host`). Please reflect this with the server changes.